### PR TITLE
chore: tidy comments in values and readme

### PR DESCRIPTION
**Pull request by STRRL**
Signed-off-by: STRRL <str_ruiling@outlook.com>

### What problem does this PR solve?

It's a part of #3015 

### What's changed and how it works?

- tidy/sync up/reorder comments in helm chart `values.yaml` and `README.md`
- remove two unused helm values:
  - `dashboard.env.DATABASE_SECRET`
  - `dnsServer.runtime`

### Related changes

- [ ] Need to update `chaos-mesh/website`
- [ ] Need to update `Dashboard UI`
- Need to **cheery-pick to release branches**
  - [ ] release-2.1
  - [ ] release-2.0

### Checklist

Tests

- [ ] Unit test
- [ ] E2E test
- [x] No code
- [ ] Manual test (add steps below)

Side effects

- [ ] Breaking backward compatibility

### Release note 

```text
Please add a release note.

You can safely ignore this section if you don't think this PR needs a release note.
```

### DCO

If you find the DCO check fails, please run commands like below (Depends on the actual situations. For example, if the failed commit isn't the most recent) to fix it:

```shell
git commit --amend --signoff
git push --force
```

### CODE REVIEW

**Comment by STRRL** [comment]
PTAL

/cc @FingerLeader @YangKeao @cwen0 @WangXiangUSTC @iguoyr

**Review by YangKeao — APPROVED** [review]
LGTM

File: helm/chaos-mesh/values.yaml [hunk 1]
```diff
@@ -283,20 +304,20 @@ dashboard:
...
 
     # If you'd like to use a DB other than SQLite (the default), set a driver + DSN here.
     DATABASE_DRIVER: sqlite3
+    # The db dsn used for Chaos Dashboard
     DATABASE_DATASOURCE: /data/core.sqlite
 
-    # If you are going to store build secrets in the Chaos Dashboard database, it is suggested that
-    # you set a database encryption secret. This must be set before any secrets are stored
-    # in the database.
-    # DATABASE_SECRET:
```
Comment (cwen0) [inline helm/chaos-mesh/values.yaml:298]: I think this feature should be provided.

Comment (cwen0) [reply helm/chaos-mesh/values.yaml:298]: I will implement it in a single pr.

File: helm/chaos-mesh/README.md [hunk 2]
```diff
@@ -18,73 +18,92 @@ The following tables list the configurable parameters of the Chaos Mesh chart an
...
-| `clusterScoped`                            | whether chaos-mesh should manage kubernetes cluster wide chaos.Also see rbac.create and controllerManager.serviceAccount | `true` |
-| `rbac.create` |  | `true`                                                |
+| `customLabels` | Customized labels that will be tagged on all the resources of Chaos Mesh | `{}` |
+| `clusterScoped` | Whether chaos-mesh should manage kubernetes cluster wide chaos.Also see rbac.create and controllerManager.serviceAccount | `true` |
+| `rbac.create` | Creating rbac API Objects. Also see clusterScoped and controllerManager.serviceAccount | `true`                                                |
 | `timezone` | The timezone where controller-manager, chaos-daemon and dashboard uses. For example: `UTC`, `Asia/Shanghai` | `UTC` |
 | `enableProfiling` | A flag to enable pprof in controller-manager and chaos-daemon  | `true` |
+| `enableCtrlServer` | A flag to enable ctrlserver which provides service to chaosctl in controller-manager. | `true` |
+| `images.registry` | the global container registry for the images, you could replace it with your self-hosted container registry. | `ghcr.io` |
```
Comment (iguoyr) [inline helm/chaos-mesh/README.md:27]: How about starting with a capital letter uniformly?

File: helm/chaos-mesh/values.yaml (hunk 1 above)
Comment (cwen0) [reply helm/chaos-mesh/values.yaml:298]: Sorry, I misunderstood the meaning of this field. At present, the data in Chaos Dashboard has no sensitive data, so this function is not necessary at present.

File: helm/chaos-mesh/README.md (hunk 2 above)
Comment (STRRL) [reply helm/chaos-mesh/README.md]: Updated PTAL

**Review by cwen0 — APPROVED** [review]
LGTM

**Pull request merged**